client/src/ui/molecules/document-survey/surveys.ts

```diff
@@ -100,4 +90,14 @@ export const SURVEYS: Survey[] = [
     start: 0,
     end: Infinity,
   },
+  {
+    key: SurveyKey.HOMEPAGE_FEEDBACK_2024,
+    bucket: SurveyBucket.HOMEPAGE_FEEDBACK_2024,
+    show: (doc: Pick<Doc, "mdn_url">) => /[^/]+\/$/i.test(doc.mdn_url),
+    src: "https://survey.alchemer.com/s3/8075407/MDN-Homepage-Improvements",
+    teaser: "We are refreshing our homepage and would love",
+    question: "your input",
```

Comment:
Nit: Usually we put the first question of the survey in the `question` field.

Reply:
@fiji-flo Is this ok for screen readers?

<img width="1069" alt="image" src="https://github.com/user-attachments/assets/51aa2606-7eb7-4a4a-9c7c-40ad5552f512">